ibex_configs.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Ibex configurations files, holds the parameter sets that are tested under CI.
+# Each configuration must specify the same set of parameters
+
+# Two-stage pipeline without additional branch target ALU and 3 cycle multiplier
+# (4 cycles for mulh), resulting in 2 stall cycles for mul (3 for mulh)
+small-3cmult:
+  RV32E                    : 0
+  RV32M                    : 1
+  BranchTargetALU          : 0
+  WritebackStage           : 0
+  MultiplierImplementation : "fast"
+
+# Three-stage pipeline with additional branch traget ALU and 1 cycle multiplier
+# (2 cycles for mulh) so mul does not stall (mulh stall 1 cycles). This is the
+# maximum performance configuration.
```

Comment:
Possible follow-up: put this information into the YAML itself, e.g. into a `description` field (or `_description` to avoid name clashes) to make it available from the ibex_config.py script. (No need to do that as part of this PR though, but please file a follow-up. We can then also use this information to auto-generate parts of the docs for supported configurations.)

Reply:
Good idea, could do it as part of a documentation PR